solanaj/src/main/java/org/p2p/solanaj/core/Account.kt

```diff
@@ -20,18 +16,29 @@ class Account {
         keyPair = TweetNaclFast.Signature.keyPair()
     }
 
-    constructor(secretKey: ByteArray) {
-        keyPair = TweetNaclFast.Signature.keyPair_fromSecretKey(secretKey)
+    constructor(keyPair: ByteArray) {
+        this.keyPair = TweetNaclFast.Signature.keyPair_fromSecretKey(keyPair)
     }
 
     private constructor(keyPair: TweetNaclFast.Signature.KeyPair) {
         this.keyPair = keyPair
     }
 
+    /**
+     * Returns an encoded [keyPair] in Base58
+     * */
+    fun getEncodedKeyPair(): String =
+        Base58Utils.encode(keypair)
+
     val publicKey: PublicKey
         get() = PublicKey(keyPair.publicKey)
 
-    val secretKey: ByteArray
+    /**
+     * The secret key is 32 byte array
+     *
+     * The keypair is 64 byte array which consists of encoded secret key and public key
```

Comment:
> the keypair is 64 byte array which consists of encoded secret key and public key.  

but you still return only secretKey from this keypair, not keypair itself

Reply:
This is the description of keypair, not of the method itself and what it returns

Reply:
This is the description of keypair, not of the method itself and what it returns